src/gui/statusbar.cpp

```diff
@@ -147,6 +158,7 @@ StatusBar::StatusBar(QWidget *parent)
     m_DHTLbl->setVisible(session->isDHTEnabled());
```

Comment:
I think the easiest way is to simply call `optionsSaved()` somewhere here.
Or extract the label visibility update into a separate method for cleaner code.